src/simple/room/Room.js

```diff
@@ -128,21 +128,31 @@ class Room extends Emitter {
    * @param constraints string
    * @param audioOnly boolean
    * @param extra Object
+   * @param room ?Room
    * @returns {Promise<Room>}
    */
-  static async connect({ extension, constraints, audioOnly = false, extra }: Object) {
+  static async connect({ extension, constraints, audioOnly = false, extra, room }: Object) {
     logger.info('connecting to room', { extension, audioOnly });
```

Comment:
You may want to log `room` (or elements of it) here

Reply:
done